crates/starknet-devnet/tests/common/background_devnet.rs

```diff
@@ -362,6 +362,10 @@ impl BackgroundDevnet {
         self.send_custom_rpc("devnet_getConfig", json!({})).await.unwrap()
     }
 
+    pub async fn get_dump(&self) -> serde_json::Value {
+        self.send_custom_rpc("devnet_dump", json!({})).await.unwrap()
+    }
+
```

Comment:
We don't need this method if it's used in just one place. Or if we really want to keep it, it could return a `Result<...>`

Reply:
Will remove it, just introduced it for consistency.